app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,17 @@
+class UserMailer < ApplicationMailer
+  default from: 'austin@wantable.com'
```

Comment:
if the default from is set up in the application_mailer you shouldn't need to do it in here too (unless you want to override it)